PTGLdynamics/PTGLdynamics.py

```diff
@@ -20,13 +20,17 @@
 import shutil
 import re
 import time
+import fnmatch
 
 
```

Comment:
Regarding line 9*: Please increment minor version number (and btw remove the TODO comment, thanks). 

* cannot comment what is out of scope of changes here